Set4/30/calculator/verifyDivision.cc

```diff
@@ -0,0 +1,13 @@
+#include "calculator.ih"                                                       
+                                                                               
+    // by expression.cc                                                        
+                                                                               
+bool Calculator::verifyDivision()
+{
+    if ((d_rhs < 0 ? -d_rhs : d_rhs) < 1e-8) // 1e-8 is the zero-cutoff
```

Comment:
```suggestion
    if (fabs(d_rhs)) // 1e-8 is the zero-cutoff
```

Also requires `include <math.h>`